rest/api.go

```diff
@@ -0,0 +1,80 @@
+package rest
+
+import (
+	"github.com/Peripli/service-manager/pkg/filter"
+	"github.com/Peripli/service-manager/pkg/plugin"
+	"github.com/sirupsen/logrus"
+)
+
+// API is the primary point for REST API registration
+type API struct {
+	// Controllers contains the registered controllers
+	Controllers []Controller
+
+	// Filters contains the registered filters
+	Filters []filter.Filter
+}
+
+// RegisterControllers registers a set of controllers
+func (api *API) RegisterControllers(controllers ...Controller) {
+	for _, controller := range controllers {
+		if controller == nil {
+			logrus.Panicln("Cannot add nil controllers")
```

Comment:
Probably you should call "continue" here to avoid nil controller to be added.

Reply:
`Panicnl` will panic and stop the server.